src/Diagnostics.RuntimeHost/Program.cs

```diff
@@ -13,12 +17,26 @@ public static void Main(string[] args)
 
         public static IWebHost BuildWebHost(string[] args)
         {
-            var config = new ConfigurationBuilder()
-                .AddCommandLine(args)
-                .Build();
-
             return WebHost.CreateDefaultBuilder(args)
-                .UseConfiguration(config)
+                .ConfigureAppConfiguration((context, config) =>
+                {
+                    var builtConfig = config
+                        .AddCommandLine(args)
+                        .Build();
+
+                    if (context.HostingEnvironment.IsDevelopment())
```

Comment:
So this code path is only for dev mode and not for Prod?

Reply:
yes, this change is for dev mode I intend to follow up with a change for prod